ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/services/ApiSubmissionService.scala

```diff
@@ -94,10 +98,12 @@ class ApiSubmissionService private (
     validation.fold(
       Future.failed,
       _ => {
-        logger.debug(
-          "Received composite command {} with let {}.",
-          if (logger.isTraceEnabled()) commands else commands.commandId.unwrap,
-          commands.ledgerEffectiveTime: Any)
+        val cmdId = commands.commandId.unwrap
+        val let = commands.ledgerEffectiveTime.toString
+        logger.debug("Received composite command with id {} and let {}.", cmdId, let)
+        logger.whenTraceEnabled {
```

Comment:
I'm slightly leaning towards using plain `slf4j` when possible. I've seen many scala logging frameworks/libraries come and go, but slf4j seems to be here to stay.

Reply:
I hear you. Should be an easy change.

Reply:
done